spec/module.dd

```diff
@@ -31,8 +31,6 @@ $(GNAME DeclDef):
     $(GLINK2 version, DebugSpecification)
     $(GLINK2 version, VersionSpecification)
     $(GLINK2 version, StaticAssert)
-    $(GLINK2 template, TemplateDeclaration)
-    $(GLINK2 template-mixin, TemplateMixinDeclaration)
     $(GLINK2 template-mixin, TemplateMixin)
     $(GLINK MixinDeclaration)
```

Comment:
TemplateMixin and MixinDeclaration can be moved to Declaration too.

Reply:
Moving MixinDeclaration to Declaration could create an ambiguity with MixinStatement.
But it should be possible to move TemplateMixin if it is also removed from NonEmptyStatementNoCaseNoDefault.

Reply:
OK, thanks.

Reply:
Now moved TemplateMixin: #3660